lib/shopify_api/resources/assigned_fulfillment_order.rb

```diff
@@ -0,0 +1,8 @@
+module ShopifyAPI
+  class AssignedFulfillmentOrder < Base
+    def self.find(scope, *args)
+      assigned_fulfillment_orders = super(scope, *args)
+      assigned_fulfillment_orders.map { |afo| FulfillmentOrder.new(afo.attributes) }
```

Comment:
Wont this cause complications? You find assigned fulfilments and then if you update that FulfillmentOrder it will hit the fulfillments endpoint. Why are you returning FulfillmentOrders from AssignedFulfillmentOrder. (this may just be me not understanding the work that you are doing)

Reply:
`AssignedFulfillmentOrders` are just `FulfillmentOrders` that just happen to be assigned. We didn't want to create two different ActiveResource types for subsequent actions common to both `AssignedFulfillmentOrders` and `FulfillmentOrders` so we convert the first to the second.